scripts/setup_test.py

```diff
@@ -143,6 +143,33 @@ def mock_print(msg_list):
             setup.test_python_version()
         self.assertEqual(print_arr, [])
 
+    def test_python_version_testing_with_incorrect_version_and_windows_os(self):
+        print_arr = []
+        def mock_print(msg_list):
+            print_arr.extend(msg_list)
+
+        print_swap = self.swap(
+            common, 'print_each_string_after_two_new_lines', mock_print)
+        uname_swap = self.swap(common, 'OS_NAME', 'Windows')
+        version_info = collections.namedtuple(
+            'version_info', ['major', 'minor'])
+        version_swap = self.swap(
+            sys, 'version_info', version_info(major=3, minor=4))
+        with print_swap, uname_swap, version_swap, self.assertRaises(Exception):
+            setup.test_python_version()
+        self.assertEqual(
+            print_arr, [
+                'It looks like you are using Windows. If you have Python '
+                'installed,',
+                'make sure it is in your PATH and that PYTHONPATH is set.',
+                'If you have two versions of Python (ie, Python 2.7 and 3), '
+                'specify 2.7 before other versions of Python when setting the '
+                'PATH.',
+                'Here are some helpful articles:',
+                'http://docs.python-guide.org/en/latest/starting/install/win/',
+                'https://stackoverflow.com/questions/3701646/how-to-add-to-the-'
+                'pythonpath-in-windows-7'])
```

Comment:
Move the contents of line 164 to line 163 and move the overflow to the next line. We want to make sure looks like an entire block for readability.

Reply:
I think that move L164 to L163 will actually make it unreadable since it is a list:

```
'It looks like you are using Windows. If you have Python '
'installed,', 'make sure it is in your PATH and that PYTHONPATH is set.',
'If you have two versions of Python (ie, Python 2.7 and 3), specify 2.7 '
'before other versions of Python when setting the PATH.',       
```

Reply:
What is there is fine. I think my comment went on an old commit.